dns/unbound-plus/src/opnsense/service/templates/OPNsense/Unboundplus/dot.conf

```diff
@@ -0,0 +1,9 @@
+{%   if helpers.exists('OPNsense.unboundplus.miscellaneous.dotservers') and OPNsense.unboundplus.miscellaneous.dotservers != '' %}
+server:
+forward-zone:
+name: "."
+forward-ssl-upstream: yes
```

Comment:
do we not need the certificate bundle reference as well?

Reply:
> do we not need the certificate bundle reference as well?

I think also this is needed, because in my manual config it was needed to configure the bundle to get DoT working:
```
tls-cert-bundle: /etc/ssl/cert.pem
```

Reply:
I made it optional but default enabled since when you drive an own instance this root ca wouldn't be in cert.pem. Hope this is fine for all